README.md

```diff
@@ -326,6 +338,28 @@ function Profile () {
 }
 ```
 
+### SSR with Next.js
+
+With the the `initialData` option, you pass an initial value to the hook. It works perfectly with many SSR solutions
```

Comment:
```suggestion
With the `initialData` option, you pass an initial value to the hook. It works perfectly with many SSR solutions
```

Reply:
Classic! I miss the double 'the' every time.